apps/ledger-live-mobile/.eslintrc.js

```diff
@@ -13,7 +14,7 @@ module.exports = {
       },
     },
   },
-  plugins: ["prettier", "detox"],
+  plugins: ["prettier", "detox", "@typescript-eslint"],
```

Comment:
This should not be needed when using the recommended rules config: https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/eslint-plugin#usage
```suggestion
  plugins: ["prettier", "detox"],
```